infra/cmake/packages/LibnpySourceConfig.cmake

```diff
@@ -0,0 +1,22 @@
+function(_LibnpySource_import)
+  if(NOT ${DOWNLOAD_LIBNPY})
+    set(LibnpySource_FOUND FALSE PARENT_SCOPE)
+    return()
+  endif(NOT ${DOWNLOAD_LIBNPY})
+
+  nnas_include(ExternalSourceTools)
+  nnas_include(OptionTools)
+
+  envoption(EXTERNAL_DOWNLOAD_SERVER "https://github.com")
+  envoption(LIBNPY_URL ${EXTERNAL_DOWNLOAD_SERVER}/llohse/libnpy/archive/refs/tags/v0.1.0.tar.gz)
+
+  ExternalSource_Download(LIBNPY
+    DIRNAME LIBNPY
+    URL ${LIBNPY_URL})
```

Comment:
why are `DIRNAME LIBNPY` and `URL` needed?

Reply:
The Libnpy required for 'q-implant' is an external library source that does not yet exist in ONE. Therefore, download is required.

'DINAME LIBNPY' and 'URL' are used to download external sources from [other cmake file](https://github.com/Samsung/ONE/blob/master/infra/cmake/packages/AbseilSourceConfig.cmake#L13).

Therefore, I thought I could refer to it and use it.

Reply:
why I asked is that other lib files didn't seem to have this. plz check othe files and it would be better to follow like oyher filed.

Reply:
Thank you. I checked and found that it works without using "DINAME LIBNPY" and "URL". Fixed and committed again.